### PR TITLE
[MIG] 12.0 mig hw telium payment terminal 

### PR DESCRIPTION
**Pull request by fmdl**
The 8.0 version doesn't work with PosBOX 19.07.

@alexis-via

### CODE REVIEW

**Comment by fmdl** [comment]
cc @sla-subteno-it

**Comment by fmdl** [comment]
cc 
@azmeuk
@vco-subteno-it

File: hw_telium_payment_terminal/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# -*- encoding: utf-8 -*-
```
Comment (azmeuk) [inline hw_telium_payment_terminal/__init__.py]: Odoo 12 requires python 3.5 so the `encoding: utf-8` header is not needed.

**Review by azmeuk — APPROVED** [review]
LGTM.

**Review by alexis-via — APPROVED** [review]

**Pull request merged**